test/blocks/slideshare/mocks/utils.js

```diff
@@ -22,6 +22,9 @@ export function createTag(tag, attributes, html) {
       el.setAttribute(key, val);
     });
   }
+  if (options?.parent) {
```

Comment:
options will always be defined, no need for `?`. I think we can skip the whole if clause and do `options.parent?.append(el);`

Reply:
yes, forgot to cleanup that test, thanks